docs/cluster-configuration.md

```diff
@@ -806,6 +806,12 @@ Possible values:
 `LiveMigrate` is the default behaviour.
 
 
+## VM state storage class
```

Comment:
The new field documentation, says that the storage class must support rwx. Let's add it here too.

Also, please add an example yaml.

Reply:
Done, PTAL!
Thanks for the review